libraries/botframework-schema/etc/botframework-schema.api.md

```diff
@@ -1614,6 +1658,18 @@ export interface TabSuggestedActions {
     actions: CardAction[];
 }
 
+// @public
+export interface TargetedMeetingNotification extends MeetingNotificationBase<TargetedMeetingNotificationInfo> {
+    channelData?: MeetingNotificationChannelData;
+    type: 'targetedMeetingNotification';
+}
+
+// @public
+export interface TargetedMeetingNotificationInfo {
```

Comment:
Can we rename the interface to `TargetedMeetingNotificationValue`.
It will align the name in existing repos.

Reply:
Will do in the in the next iteration